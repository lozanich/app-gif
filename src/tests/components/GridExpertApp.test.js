import "@testing-library/jest-dom";
import React from "react";
import GridExpertApp from "../../components/GifExpertApp";
import { shallow } from "enzyme";

describe("Pruebas al componente GridExpertApp", () => {
  const wrapper = shallow(<GridExpertApp />);

  test("Deberia de ser el snapshot de GridExpertApp", () => {
    expect(wrapper).toMatchSnapshot();
  });

  test("Deberia de mostarr una lista de categorias", () => {
    const categories = ["Goku", "One punch"];
    const wrapper = shallow(<GridExpertApp defaultCategories={categories} />);

    expect(wrapper.find("GifGrid").length).toBe(categories.length);
  });
});
