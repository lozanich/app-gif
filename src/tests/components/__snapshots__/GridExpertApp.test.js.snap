// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas al componente GridExpertApp Deberia de ser el snapshot de GridExpertApp 1`] = `
<Fragment>
  <h2>
    Gif expert app
  </h2>
  <hr />
  <AddCategory
    setCategories={[Function]}
  />
  <ol />
</Fragment>
`;
